gpflow/optimizers/mcmc.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2019 Artem Artemev @awav
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from typing import Callable, List, Optional, TypeVar
+
+import tensorflow as tf
+
+from gpflow.base import Parameter, positive
+
+__all__ = ["positive_parameter", "SamplingHelper"]
+
+
+def positive_parameter(value: tf.Tensor):
```

Comment:
It seems like this shouldn't be in `mcmc.py` but better belongs somewhere else...?

Reply:
i would agree with you, but i cant comment too much but i assumed there was a reason, we can ask @awav